include/seastar/core/metrics_api.hh

```diff
@@ -41,7 +41,7 @@ namespace seastar {
 namespace metrics {
 namespace impl {
 
-using labels_type = std::map<sstring, sstring>;
+using labels_type = std::map<sstring, label_value>;
```

Comment:
I believe that's exposed as an API and will break scylladb unit tests that verify metrics.

Reply:
I attempted to avoid changing this in place now: https://github.com/scylladb/seastar/pull/2400#issuecomment-2340443138